chia/wallet/puzzles/augmented_condition.clvm

```diff
@@ -0,0 +1,13 @@
+(mod
+  (
+    CONDITION
+    INNER_PUZZLE
+    inner_solution
+  )
+
+  (c
+    CONDITION
+    (a INNER_PUZZLE inner_solution)
+  )
+
+)
```

Comment:
You don't have to change this (can just resolve this conversation as soon as you see it) but this is such a small program that it's probably not worth its own file because it compiles down to just `(c 2 (a 5 11))` which is probably better as just a `Program.to(...)` call with a comment explaining what it does.

Reply:
I didn't change any chialisp code. Maybe @geoffwalmsley can answer this.